Core/ReadInput/InputReader.cs

```diff
@@ -29,24 +30,40 @@ public class InputReader
   private const string c_moreChoicesText = "[grey](Move up and down to choose version)[/]";
 
   private readonly IAnsiConsole _console;
+  private readonly Style _textPromptStyle;
+  private readonly Style _selectionPromptHighlightStyle;
+  //standard console colors, #10 & #9 respectively
+  private readonly string _yesColor = "lime";
+  private readonly string _noColor = "red";
```

Comment:
Could you check, please, if the parser only supports the lower-case versions?
https://spectreconsole.net/appendix/colors

From reading the code, I think it doesn't actually work the way you intend it? (demo?) My reading is that if you specify the lower-case color, it will use that color and not map it to the Console-position. If that holds true, we might just skip the colors alltogether.

Reply:
This https://github.com/spectreconsole/spectre.console/issues/325 could indicate that uppercase does in fact work.

Reply:
And okay, https://github.com/spectreconsole/spectre.console/pull/409/files#diff-7f4a6c8fdcc7b50c32fc4bd91564fc43bd241d88a674a788d4b7dd81aaa76fcaR31-R32 indicates that `red` and `lime` are okay for our use case. only one way to find out :) (Demo!)

Reply:
I should've probably made a comment before about how I already ran a demo ;) 

When changing the standard colors in powershell, it also changes the color for the confirmation